backend/src/services/classes/insertLike.js

```diff
@@ -0,0 +1,31 @@
+const insertLike = async (req, res) => {
```

Comment:
```suggestion
export async function insertLike (req, res) {
```

이렇게 바꾸고 맨 밑에 `export { insertLike };` 는 지워도 됩니당

Reply:
deleteLike 도 마찬가지에유

Reply:
classesList도 동일하게 수정해야 할 것 같아서 수정했는데 맞겠죠?ㅎㅎ